ush/emcsfc_snow.sh

```diff
@@ -100,12 +97,9 @@ GFS_LONSPERLAT_FILE=${GFS_LONSPERLAT_FILE:-global_lonsperlat.t1534.3072.1536.txt
 
 #------------------------------------------------------------------------
 # Input snow data.  ims snow cover and afwa snow depth. ims is NH only.
-# In OPS, we run with ims only, or ims and afwa.  afwa data is too
-# unreliable to use on its own.  ims is grib2.  afwa is grib1.
+# AFWA is global.
 #------------------------------------------------------------------------
 
-AFWA_NH_FILE=${AFWA_NH_FILE:-"NPR.SNWN.SP.S1200.MESH16"}
-AFWA_SH_FILE=${AFWA_SH_FILE:-"NPR.SNWS.SP.S1200.MESH16"}
 AFWA_GLOBAL_FILE=${AFWA_GLOBAL_FILE:-""}
```

Comment:
No default setting for `AFWA_GLOBAL_FILE`?

Reply:
Good catch. I will add one.